### ISSUE TITLE
Fix common errors - add option to remove dialog dashes in multiple lines?

### ISSUE THREAD

**Issue by Toxic-Waste-** [SubtitleEdit/subtitleedit#4441]
Right now the common fixes have a fix to Remove dialog dashes in single lines. Is it possible to also include a variant to do this for mulitple lines? 

Maybe I am overlooking the option, but I have tried every 'dash fixer' I can find on certain subtitles and I still end with things like:

```
- I have to...
take a rain check on this game.
```

Or

```
- All right! Don't.
Don't look. Don't look.
```

As the dash is completely redundant, it makes sense to strip them, but I cant seem to find the option?

**Comment by OmrSi** [comment]
Aside from a proper option in the settings, you can use this Regex if you want a quick fix:
![image](https://user-images.githubusercontent.com/20923700/96284820-cc55c580-0fe6-11eb-977e-ca50e8562403.png)

`\A-\s?(.*)(\n(?!-).*)?\z` with `$1$2`

**Comment by Toxic-Waste-** [comment]
thanks for the quick fix, works like a charm! Hard to implement this in the FCE?

**Comment by niksedk** [comment]
Fixed in fbc2d15912add75c3e1ed890f0575b7143ead814
